### PR TITLE
Remove code comment

### PR DESCRIPTION
**Pull request by dgbarclay**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Fix failing test. Change to code is on line 625 of notebook diff.

### CODE REVIEW

File: examples/entitlements/Access Metadata entitlements.ipynb [hunk 1]
```diff
@@ -714,7 +625,7 @@
     "    type=\"Personal\"\n",
     ")\n",
     "\n",
-    "#users_api.create_user(create_user_request)"
+    "users_api.create_user(create_user_request)"
```
Comment (dgbarclay) [inline examples/entitlements/Access Metadata entitlements.ipynb:628]: This is the only change to code

File: examples/entitlements/Access Metadata entitlements.ipynb [hunk 2]
```diff
@@ -2,38 +2,12 @@
  "cells": [
```
Comment (rizwansaeed) [inline examples/entitlements/Access Metadata entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #9.    users_api.create_user(create_user_request)
</pre><p> </p><p>does this actually create the user or does it need some external action e.g. someone going to the website to approve?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (rizwansaeed) [inline examples/entitlements/Access Metadata entitlements.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #9.    users_api.create_user(create_user_request)
</pre><p> </p><p>also, what happens on the 2nd run? i suspect we may need a <code>try-except</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (dgbarclay) [reply examples/entitlements/Access Metadata entitlements.ipynb:2]: It creates them but in a 'Provisioned' state, for the notebook tho I _think_ that is okay.

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**